README.md

```diff
@@ -1,6 +1,8 @@
 Converge API PHP class
 ======================
 
+[![Build Status](https://travis-ci.org/israelshirk/converge-api-php.svg?branch=master)](https://travis-ci.org/israelshirk/converge-api-php)
```

Comment:
I have created a Travis CI page for this at https://travis-ci.org/markroland/converge-api-php -- let's use that.